pulse2percept/tests/test_api.py

```diff
@@ -53,7 +53,7 @@ def test_Simulation_set_optic_fiber_layer():
     x_range = (-100, 100)
     y_range = (0, 200)
     sim.set_optic_fiber_layer(x_range=x_range, y_range=y_range,
-                              save_data=False)
+                              save_data=False, alpha = 14000)
```

Comment:
It would be good to add a `npt.assert_equal` statement to make sure it works as intended.